frontend/ChapterList.vue

```diff
@@ -415,19 +420,51 @@
                   icon="phone"
                 ></b-input>
               </b-table-column>
-              <b-table-column field="Facebook" label="Facebook" v-slot="props">
-                <b-input
-                  type="text"
-                  v-model="props.row.Facebook"
-                  placeholder="Facebook"
-                  icon="facebook"
-                ></b-input>
-              </b-table-column>
               <b-table-column v-slot="props">
                 <b-button @click="deleteOrganizer(props.row)">
                   <b-icon icon="delete" type="is-danger"></b-icon>
                 </b-button>
               </b-table-column>
+              <template #detail="props">
+                <div style="display: flex; flex-wrap: wrap; gap: 10px">
+                  <b-field label="Facebook" label-position="on-border" style="max-width: 300px">
+                    <b-input
+                      type="text"
+                      v-model="props.row.Facebook"
+                      placeholder="Facebook"
+                      icon="facebook"
+                      maxlength="100"
+                    ></b-input>
+                  </b-field>
+                  <b-field label="Instagram" label-position="on-border" style="max-width: 300px">
```

Comment:
i wonder if we should do this style of labels for the "name email phone" fields too to keep it a bit more consistent?

Reply:
the table already has a header row that labels the columns themselves so individual labels aren't really necessary in table cells. lmk if i'm missing something though. and max-width doesn't matter as much since the table already forces 3 textboxes to share one line so those elements are never too wide.